### PR TITLE
Adding changelog for 8.11.1.

### PR DESCRIPTION
**Pull request by ppedrot**
Like the *Port-Salut*. Can we get that in fast to release 8.11.1? Thanks.

### CODE REVIEW

**Comment by Zimmi48** [comment]
@ppedrot Nope, I just didn't work hard enough on the script to have them removed automatically.

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -647,6 +647,79 @@ Changes in 8.11.0
...
+**Kernel**
+
+- **Fixed:**
+  Allow more inductive types in `Unset Positivity Checking` mode
+  (`#11811 <https://github.com/coq/coq/pull/11811>`_,
+  by SimonBoulier).
+
+**Specification language, type inference**
+
+
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:662]:
```suggestion
```

File: doc/sphinx/changes.rst [hunk 2]
```diff
@@ -647,6 +647,79 @@ Changes in 8.11.0
...
+**Tactic language**
+
+
+**Commands and options**
+
+
+**Tools**
+
+
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:691]:
```suggestion
```

File: doc/sphinx/changes.rst [hunk 3]
```diff
@@ -647,6 +647,79 @@ Changes in 8.11.0
...
+**CoqIDE**
+
+- **Fixed:**
+  Compiling file paths containing spaces
+  (`#10008 <https://github.com/coq/coq/pull/10008>`_,
+  by snyke7, fixing `#11595 <https://github.com/coq/coq/pull/11595>`_).
+
+**Standard library**
+
+
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:701]:
```suggestion
```

File: doc/sphinx/changes.rst [hunk 4]
```diff
@@ -647,6 +647,79 @@ Changes in 8.11.0
...
+  Bump official OCaml support and CI testing to 4.10.0
+  (`#11131 <https://github.com/coq/coq/pull/11131>`_,
+  `#11123 <https://github.com/coq/coq/pull/11123>`_,
+  `#11102 <https://github.com/coq/coq/pull/11123>`_,
+  by Emilio Jesus Gallego Arias, Jacques-Henri Jourdan,
+  Guillaume Melquiond, and Guillaume Munch-Maccagnoni).
+- **Added:**
+  Bump official OCaml support to 4.09.1
+  (`#11860 <https://github.com/coq/coq/pull/11860>`_,
+  by Emilio Jesus Gallego Arias).
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:714]: I suggest removing this one because it is made obsolete by the one above:
```suggestion
```

**Review by Zimmi48 — COMMENTED** [review]
~~You can apply a batch of suggestion from the "Files changes" tab.~~ Never mind.

**Comment by ppedrot** [comment]
The miracle of race conditions. Anyways, comments integrated.

**Review by Zimmi48 — APPROVED** [review]
That's a pretty quick feedback loop!

**Comment by Zimmi48** [comment]
I'll merge as soon as the CI has built the refman.

**Comment by ejgallego** [comment]
So what happened with the problems with native compute?

I saw this strange code that makes Coq segfault if the flag is not set, however the bugs that the code was supposed to address are still open?

**Pull request merged**